packages/form-field/src/FormFieldFeedback/FormFieldFeedback.styles.ts

```diff
@@ -0,0 +1,27 @@
+import { css } from '@leafygreen-ui/emotion';
+import { Size, spacing, typeScales } from '@leafygreen-ui/tokens';
+
+export const containerStyles = css`
+  display: flex;
+  gap: ${spacing[100]}px;
+`;
+
+export const spacingTop = css`
+  padding-top: ${spacing[100]}px;
+`;
+
+export const hideContainerStyle = css`
+  opacity: 0;
+`;
```

Comment:
this is conditionally added for cases where a popover may render below an input container but not fully cover the feedback message. adding opacity: 0 will effectively erase the feedback when the menu is open